pom.xml

```diff
@@ -163,8 +161,10 @@
             <parallelThreads>2</parallelThreads>
             <properties>
               <!-- maven-compiler-plugin properties -->
-              <maven.main.skip>${maven.main.skip}</maven.main.skip>
-              <maven.compiler.useIncrementalCompilation>false</maven.compiler.useIncrementalCompilation>
+              <maven.main.skip>false</maven.main.skip>
```

Comment:
I think this might stay the property to be able to override the behavior easily.

Reply:
Thank you very much for the review. Agreed, I'm going to undo this change.

Reply:
properties were restored in the root pom.xml